EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -98,6 +105,21 @@ public struct MediaPlayerView: View {
                 handleControl(control)
             }
     }
+
+    @ViewBuilder private func renderTimeBar(color: Color = .blue) -> some View {
+        Slider(value: $viewModel.playerCurrentTime, in: 0...viewModel.playerDuration){
+            // No label needed for now
+        } minimumValueLabel: {
+            Text(Util.Date.timeFormat(viewModel.playerCurrentTime, totalTimeInSeconds: viewModel.playerDuration))
+                .foregroundColor(.white)
+                .font(.system(size: Constants.timeFontSize))
+        } maximumValueLabel: {
+            Text(Util.Date.timeFormat(viewModel.playerDuration, totalTimeInSeconds: viewModel.playerDuration))
+                .foregroundColor(.white)
+                .font(.system(size: Constants.timeFontSize))
+        }
+        .tint(color)
```

Comment:
I don't think we should use the native Slider control here, it's barely customizable. Although this UI will probably not even be used for most people that use the library, so I'm not sure. Thoughts? @JPedroAmorim, @pedromotita, @ricardodalarme